apps/web/src/components/Settings/Manager/ProfileManager/Management/List.tsx

```diff
@@ -0,0 +1,153 @@
+import type {
+  LastLoggedInProfileRequest,
+  Profile,
+  ProfilesManagedRequest
+} from '@hey/lens';
+
+import Loader from '@components/Shared/Loader';
+import UserProfile from '@components/Shared/UserProfile';
+import { UsersIcon } from '@heroicons/react/24/outline';
+import {
+  ManagedProfileVisibility,
+  useHideManagedProfileMutation,
+  useProfilesManagedQuery,
+  useUnhideManagedProfileMutation
+} from '@hey/lens';
+import { Button, EmptyState, ErrorMessage } from '@hey/ui';
+import errorToast from '@lib/errorToast';
+import { type FC, useEffect } from 'react';
+import { useInView } from 'react-cool-inview';
+import toast from 'react-hot-toast';
+import { useAccount } from 'wagmi';
+
+interface ListProps {
+  managed?: boolean;
+}
+
+const List: FC<ListProps> = ({ managed = false }) => {
+  const { address } = useAccount();
+
+  const lastLoggedInProfileRequest: LastLoggedInProfileRequest = {
+    for: address
+  };
+
+  const profilesManagedRequest: ProfilesManagedRequest = {
+    for: address,
+    hiddenFilter: managed
+      ? ManagedProfileVisibility.NoneHidden
+      : ManagedProfileVisibility.HiddenOnly
+  };
+
+  const { data, error, fetchMore, loading, refetch } = useProfilesManagedQuery({
+    variables: {
+      lastLoggedInProfileRequest,
+      profilesManagedRequest
+    }
+  });
+
+  const [hideManagedProfile, { loading: hiding }] =
+    useHideManagedProfileMutation();
+  const [unhideManagedProfile, { loading: unhiding }] =
+    useUnhideManagedProfileMutation();
+
+  useEffect(() => {
+    refetch();
+  }, [managed, refetch]);
+
+  const profilesManaged = data?.profilesManaged.items;
+  const pageInfo = data?.profilesManaged?.pageInfo;
+  const hasMore = pageInfo?.next;
+
+  const { observe } = useInView({
+    onChange: async ({ inView }) => {
+      if (!inView || !hasMore) {
+        return;
+      }
+
+      return await fetchMore({
+        variables: {
+          lastLoggedInProfileRequest,
+          profilesManagedRequest: {
+            ...profilesManagedRequest,
+            cursor: pageInfo.next
+          }
+        }
+      });
+    }
+  });
+
+  if (loading) {
+    return (
+      <div className="pb-5">
+        <Loader />
+      </div>
```

Comment:
why are you adding this extra div. You could also add margin to loader instead.

Reply:
Done ✅